scss/_orange-navbar.scss

```diff
@@ -0,0 +1,62 @@
+// Contents
+//
+// Supra bar
+
+
+// Supra bar
+//
+// Brand exclusive thin navbar that can be used with orange navbar
+.supra {
+  display: none;
+  padding-top: 0;
+  padding-bottom: 0;
+  border-bottom: $navbar-orange-border-bottom solid $gray-700;
+
+  @include media-breakpoint-up(md) {
+    display: flex;
+    .container {
+      padding-right: $navbar-orange-padding-x;
+      padding-left: $navbar-orange-padding-x;
+    }
+  }
+
+  .navbar-nav .nav-link {
+    padding: #{map-get($spacers, 2) + $navbar-supra-padding-y} $navbar-supra-padding-x $navbar-orange-link-padding-bottom;
+    font-size: $font-size-sm;
+    line-height: 1;
+
+    &.active::before {
+      right: $navbar-supra-padding-x;
+      bottom: -$navbar-orange-border-bottom;
```

Comment:
border-bottom -> border-bottom-width to be clear, looks strange to use a border as a bottom space ^^

Reply:
border-bottom-height, I'd say
Done

Reply:
No `width` because it's a property of the border even if at bottom it looks like a height ;-)
https://developer.mozilla.org/fr/docs/Web/CSS/border

Reply:
Done, thx